code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -108,4 +108,27 @@
 		if(silenced)
 			user.emote("me",1,"pretends to fire a gun at [target]!")
 		else
-			user.say(pick("BANG!", "BOOM!", "PEW!", "KAPOW!"))
\ No newline at end of file
+			user.say(pick("BANG!", "BOOM!", "PEW!", "KAPOW!"))
+
+/obj/item/weapon/gun/projectile/sec
+	name = "\improper NT Mk58 .45 pistol"
+	desc = "The NT Mk58 is a cheap, ubiquitous sidearm, produced by a NanoTrasen subsidiary. Uses .45 rounds."
+	icon_state = "secguncomp"
+	mag_type = "/obj/item/ammo_storage/magazine/c45"
+	caliber = ".45"
+	origin_tech = list(TECH_COMBAT = 2, TECH_MATERIAL = 2)
```

Comment:
origin_tech is a concatenated string for us, not a list. The only reason this even compiled is because Byond has the crap feature of allowing arbitrary list keys and assuming they're strings.
The tech defines are also different.